libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -7,21 +7,21 @@
       placeholder="Search for books to add to your reading list"
       formControlName="term"
     />
-    <button mat-icon-button matSuffix>
+    <button type="search" aria-label = "Search Books" mat-icon-button matSuffix>
       <mat-icon>search</mat-icon>
     </button>
   </mat-form-field>
 </form>
 
 <ng-container *ngIf="searchTerm; else empty">
   <div class="book-grid">
-    <div class="book" data-testing="book-item" *ngFor="let b of books">
-      <div class="book--title">
+    <div class="book" data-testing="book-item" *ngFor="let b of this.books$|async">
+      <div tabindex="0" class="book--title">
```

Comment:
Since this element isn't interactive, we don't want a tabindex attribute here. Screenreaders read content aloud even if it is not focusable. Actually, we hardly ever need to use tabindex="0" in our HTML. The only time we would want to use it is if we were creating a custom element that we wanted the user to be able to interact with (like if we were creating a button-like element out of divs). More info can be found here: https://developer.paciellogroup.com/blog/2014/08/using-the-tabindex-attribute/

Reply:
Very helpful Thank you :-), I have knowledge on this but over thinking caused all these :-)